src/js/themes/base.d.ts

```diff
@@ -81,10 +81,12 @@ type Colors = typeof colors & {
 
 interface ButtonKindType {
   background?: BackgroundType;
-  border?: {
-    color?: ColorType;
-    width?: string;
-  } | boolean;
+  border?:
+    | {
```

Comment:
Is the pipe in the front a new convention? noticing it changed on all of these

Reply:
I think this was the prettier I have on my local I will change it back